Entity-ngsiv2-schema.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Entity-ngsiv2-schema.json",
+  "title": "NGSIv2 Entity",
+  "description": "NGSIv2 Entity",
+  "definitions": {
+    "EntityIdentifier": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256,
+      "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$"
+    },
+    "Name": {
+      "type": "string",
+      "pattern": "^(\\d|[a-zA-Z]|_)+$",
+      "minLength": 1,
+      "description": "NGSIv2 Name"
+    },
+    "Value": {
+      "type": [
+        "number",
+        "boolean",
+        "array",
+        "object",
+        "null",
+        "string"
+      ]
+    },
+    "TypeValue": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/Name"
+            },
+            {
+              "not": {
+                "const": "DateTime"
+              }
+            }
+          ]
+        },
+        "value": {
+          "$ref": "#/definitions/Value"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DateTimeValue": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "DateTimeTypeValue": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "const": "DateTime"
+        },
+        "value": {
+          "$ref": "#/definitions/DateTimeValue"
```

Comment:
This is "ours"? Or it refers to an existing and standard JASON schema for ISO8601 formatted dates?

Reply:
see above

Reply:
You mean a comment above this one? Or a line above this one?

(Sorry a probably stupid question :) but I'm not so familiar with JSON Schema and problably I'm missing important links to external JSON spec. If it is the second option and you can tell me around which code line, great!)

Reply:
not at all :)

We have created some definitions for DateTimeValue and DateTimeTypeValue, taking advantage of the JSON Schema constructs ;) Hope this helps

https://github.com/Fiware/dataModels/pull/373/files#diff-c4a04684aa22645d0665ae4a1afc7cf0R66

Reply:
I see 

```
 "format": "date-time"
```

Is that date-time a JSON Schema construct? Do you have a link to the place where it is documented?

Reply:
https://json-schema.org/latest/json-schema-validation.html#rfc.section.7.3.1

Reply:
Thanks for the link!

NTC